app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -713,8 +761,8 @@ class ThreadAdapter(
         override val binding: ItemMessageBinding,
         private val shouldLoadDistantResources: Boolean,
         onContactClicked: ((contact: Recipient) -> Unit)?,
-        onAttachmentClicked: ((attachment: Attachment) -> Unit)?,
-        onAttachmentOptionsClicked: ((attachment: Attachment) -> Unit)?,
+        onAttachmentClicked: ((attachment: Attachable) -> Unit)?,
+        onAttachmentOptionsClicked: ((attachment: Attachable) -> Unit)?,
```

Comment:
Is tis normal we have a variable named `attachment` with type `Attachable`?

Reply:
The goal of this PR is to be able to view SwissTransfer attachments as regular attachments. The type is Attachable because it is an interface implemented by types that are attachments.

Reply:
Yeah I know. But it's a bit confusing to have an attachment of type Attachable. But if everyone is ok with that, I'm also ok.

Reply:
I'm personally not shocked by a variable attachment of type attachable